tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
     public function testExtraStylesheetsGetAdded()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $extraStylesheets = ['foo/bar.css', 'bar/quux.css'];
         $this->load([
             'assets' => [
-                'extra_stylesheets' => [
-                    'foo/bar.css',
-                    'bar/quux.css',
-                ],
+                'extra_stylesheets' => $extraStylesheets,
             ],
         ]);
         $stylesheets = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['stylesheets'];
 
-        $this->assertEquals($stylesheets, [
-            'bundles/sonatacore/vendor/bootstrap/dist/css/bootstrap.min.css',
-            'bundles/sonatacore/vendor/components-font-awesome/css/font-awesome.min.css',
-            'bundles/sonatacore/vendor/ionicons/css/ionicons.min.css',
-            'bundles/sonataadmin/vendor/admin-lte/dist/css/AdminLTE.min.css',
-            'bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css',
-            'bundles/sonataadmin/vendor/iCheck/skins/square/blue.css',
-            'bundles/sonatacore/vendor/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
-            'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css',
-            'bundles/sonatacore/vendor/select2/select2.css',
-            'bundles/sonatacore/vendor/select2-bootstrap-css/select2-bootstrap.min.css',
-            'bundles/sonataadmin/vendor/x-editable/dist/bootstrap3-editable/css/bootstrap-editable.css',
-            'bundles/sonataadmin/css/styles.css',
-            'bundles/sonataadmin/css/layout.css',
-            'bundles/sonataadmin/css/tree.css',
-            'foo/bar.css',
-            'bar/quux.css',
-        ]);
+        $this->assertSame(array_merge($this->defaultStylesheets, $extraStylesheets), $stylesheets);
     }
 
     public function testRemoveStylesheetsGetRemoved()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $removeStylesheets = ['bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css', 'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css'];
```

Comment:
It seems that his line is over 120 characters, could you please break it? Thank you